### ISSUE TITLE
DM roles plugin

### ISSUE THREAD

**Issue by tanndlin** [cs-discord-at-ucf/lion#543]
Similar to broadcast but to DM users with specific role

Obviously admin only, and similar flow to broadcast plugin

**Comment by suneettipirneni** [comment]
I don't think this would be feasible since the bot can only send one DM at time, if you have like 100 members as a part of a role it'll have to do 100 different requests to send the DM's out, which will be extremely slow, and put a lot of load on the bot.

**Comment by tanndlin** [comment]
It's not intended for huge roles, this is why it's admin only lol. Used lightly

**Comment by suneettipirneni** [comment]
Why not just do a role ping instead then?
